precompiles/assets-erc20/example/README.md

```diff
@@ -0,0 +1,92 @@
+# How to Convert Existing ERC20 Token Into XC20 Asset
+This an example how you can enhance already deployed ERC20 token to cross-chain-ready XC20 asset.
+
```

Comment:
Perhaps we should first describe the scenario at which this example is aimed at.